### PR TITLE
[openmvs] use project install folder (#10265)

### PR DESCRIPTION
**Pull request by cdcseacave**
Solves #10265

### CODE REVIEW

File: ports/openmvs/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: openmvs
-Version: 1.1
+Version: 1.1-1
 Description: OpenMVS: open Multi-View Stereo reconstruction library
```
Comment (NancyLi1013) [inline ports/openmvs/CONTROL]: Could you please add `homepage `here?
`Homepage: https://github.com/cdcseacave/openMVS`

File: ports/openmvs/portfile.cmake [hunk 2]
```diff
@@ -19,7 +19,6 @@ vcpkg_configure_cmake(
         -DINSTALL_CMAKE_DIR:STRING=share/openmvs
         -DINSTALL_BIN_DIR:STRING=bin
         -DINSTALL_LIB_DIR:STRING=lib
-        -DINSTALL_INCLUDE_DIR:STRING=include
```
Comment (NancyLi1013) [inline ports/openmvs/portfile.cmake]: `include(vcpkg_common_functions)` is no longer needed.
Could you please remove this line?

Comment (NancyLi1013) [reply ports/openmvs/portfile.cmake]: Could you please update
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/openmvs RENAME copyright)`
as
`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by cdcseacave** [comment]
@NancyLi1013 done, thank you

**Review by JackBoosY — APPROVED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Hi @cdcseacave 
Thanks for this PR.
LGTM now.

@strega-nil 
Could you please help priority to review and merge this PR?

Thanks.

**Comment by strega-nil** [comment]
Thanks so much @cdcseacave :D

**Pull request merged**